scripts/generate-releases.sh

```diff
@@ -35,33 +35,28 @@ validate_environment() {
     fi
 }
 
-# Fetch the list of snapshots for provided commit and branch values. 
-# Make sure that only one the most recent snapshot per application is returned.
-get_snapshots() {
+# Fetch the list of snapshot names and associented application name for provided commit and branch values.
+get_snapshots_data() {
     local -r commit="$1"
     local -r branch="$2"
-    kubectl get -n rh-acs-tenant snapshot -l pac.test.appstudio.openshift.io/sha="${commit}" -o json | jq -r '
-        .items
-        | map(select((.metadata.annotations["pac.test.appstudio.openshift.io/source-branch"]=="'"${branch}"'") or (.metadata.annotations["pac.test.appstudio.openshift.io/source-branch"]=="refs/heads/'${branch}'")))
-        | sort_by(.spec.application)
-        | group_by(.spec.application)
-        | map(sort_by(.metadata.creationTimestamp) | last)
-        | .[]
+    snapshot_list="$(get_snapshots "$commit" "$branch")"
+    echo "$snapshot_list" | jq -r '
+        .[]
         | "\(.metadata.name)|\(.spec.application)"'
```

Comment:
Nt: this should fit now:
```suggestion
    echo "$snapshot_list" | jq -r '.[] | "\(.metadata.name)|\(.spec.application)"'
```